### PR TITLE
Frontend: Migrating profile view from web to separate controller.

### LINKED ISSUE #737: Integrate API for getting the profile data in Client Side.
When the user is directed to `My profile` after updating the profile, then the updated changes are not visible. The changes are visible only when the user again refreshes the page.

### PR DESCRIPTION
**Pull request by RishabhJain2018**
fixes #737 . @deshraj @spyshiv @aka-jain Please review the PR.

### CODE REVIEW

File: frontend/src/js/controllers/profileCtrl.js [hunk 1]
```diff
@@ -6,13 +6,21 @@
 
     angular
         .module('evalai')
-        .controller('ProfileCtrl', ProfileCtrl);
+        .controller('profileCtrl', profileCtrl);
 
-    ProfileCtrl.$inject = ['utilities', '$state', '$stateParams'];
+    profileCtrl.$inject = ['utilities', '$state', '$stateParams', '$rootScope'];
```
Comment (spyshiv) [inline frontend/src/js/controllers/profileCtrl.js]: There is no need to inject `$state` and `$stateParams`. Just remove it.

File: frontend/src/js/controllers/profileCtrl.js [hunk 2]
```diff
@@ -6,13 +6,21 @@
...
 
-    function ProfileCtrl(utilities, $state, $stateParams) {
+    function profileCtrl(utilities, $state, $stateParams, $rootScope) {
```
Comment (spyshiv) [inline frontend/src/js/controllers/profileCtrl.js]: Same here

File: frontend/src/views/web/profile.html [hunk 3]
```diff
@@ -2,14 +2,14 @@
     <slide-profile></slide-profile>
     <div class="ev-md-container center">
         <img ng-src="{{profile.imgUrl}}" height="80" width="auto">
-        <div>{{web.user.username}}</div>
-        <div class="text-light-gray	w-300">{{web.user.email}}</div>
+        <div>{{profile.user.username}}</div>
+        <div class="text-light-gray	w-300">{{profile.user.email}}</div>
```
Comment (spyshiv) [inline frontend/src/views/web/profile.html]: A lot of space between two classes. Fix it.

File: frontend/src/views/web/profile.html [hunk 4]
```diff
@@ -19,6 +19,6 @@
                 <label for="sub">Subscribe to email list</label>
```
Comment (spyshiv) [inline frontend/src/views/web/profile.html]: Remove `for="sub"` since there is no input element having `id="sub"`.

**Comment by RishabhJain2018** [comment]
@spyshiv Please review the PR.

File: frontend/src/views/web/profile.html [hunk 5]
```diff
@@ -2,23 +2,19 @@
...
+                <span class="text-light-gray fs-12">First Name<br> </span> {{profile.user.first_name}}
             </li>
             <li>
-                <span class="text-light-gray fs-12">Last Name<br></span> {{web.user.last_name}}
+                <span class="text-light-gray fs-12">Last Name<br></span> {{profile.user.last_name}}
             </li>
             <li>
                 <span class="text-light-gray fs-12">Affiliated To<br></span> -
             </li>
-            <li>
```
Comment (aka-jain) [inline frontend/src/views/web/profile.html]: @RishabhJain2018 make this as a comment. May be in near future we need this feature.

Comment (RishabhJain2018) [inline frontend/src/views/web/profile.html]: okay sure.

**Comment by RishabhJain2018** [comment]
@spyshiv @aka-jain Please review the PR.

**Comment by aka-jain** [comment]
LGTM. Merging it.

**Pull request merged**